doc/source/coding-style/required-standard.rst

```diff
@@ -36,9 +35,9 @@ Required ``pyproject.toml`` configuration
     [tool.pydocstyle]
     convention = "numpy"
 
-
 Required ``.flake8`` configuration
 ----------------------------------
```

Comment:
```suggestion
Required Flake8 configuration
-----------------------------
```
I don't think we need to tag this general reference as a code entity. It's a pain how some of these have spaces (PEP 8) and some don't (Flake8). It's always worth looking up how they are formatted in their respective doc.